ui/partials/main/thSheriffPanel.html

```diff
@@ -12,7 +12,7 @@
 
   <!-- Exclusion profiles -->
   <div ng-if="view == 'exclusion_profile_list'"
-       class="th-inline-option-group add-new-exclusion">
```

Comment:
These two `add-new-exclusion` classes here and below didn't appear to be used since they have no child `input` elements (see L168 above which uses it for targeting). So essentially they got replaced by a new `exclusion-container` class. Other `add-new-exclusion` classes are still left in this file, which do have inputs.